Userland/Libraries/LibTLS/Certificate.cpp

```diff
@@ -349,6 +349,14 @@ Optional<Certificate> Certificate::parse_asn1(ReadonlyBytes buffer, bool)
             return {};
     }
 
+    // self signed
+    {
+        // FIXME: Actually parse out Authority Key Identifier and Subject Key Identifier to allow comparing
+        //        - Either it has matched Authority Key Identifier with Subject Key Identifier,
+        //        - in some cases there is no Authority Key identifier, then Issuer string should match with Subject string
+        certificate.is_self_signed = certificate.issuer_identifier_string() == certificate.subject_identifier_string();
```

Comment:
the condition you provided should be named `is_self_issued`. letʼs be very specific in this!

Reply:
Do you have any resource backing you on this? Everything I found only talks about the two here mentioned things.

Reply:
Would you be ok with this solution?

Reply:
definitely! added few more remarks but if it was on me—iʼd accept what we have here! 😄